conda/recipes/cucim/meta.yaml

```diff
@@ -38,21 +40,33 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake >=3.23.1,!=3.25.0
     - make
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - click
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% endif %}
     - cupy >=12.0.0
     - libcucim ={{ version }}
     - numpy 1.21
     - python
     - scikit-image >=0.19.0,<0.21.0a0
     - scipy
   run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+    - cuda-version {{ cuda_spec }}
```

Comment:
We should use pin_compatible here to match what other RAPIDS recipes have done. Then you don’t need cuda_spec.